minecraft_servers/lumiacraft/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "lumiacraft",
+  "nice_name": "Lumiacraft",
+  "direct_ip": "mc.lumiacraft.net",
+  "server_wildcards": ["%.lumiacraft.net"],
+  "supported_languages": ["en"],
+  "social": {
+    "web": "https://lumiacraft.net",
+    "web_shop": "https://lumiacraft-creative.tebex.io",
+    "web_support": "https://lumiacraft.net",
```

Comment:
Only FAQ/Contact pages allowed here.

Reply:
> Only FAQ/Contact pages allowed here.

Can you elaborate?
I think these fields are correct, here is another example of the same exact thing.
https://github.com/LabyMod/server-media/blob/master/minecraft_servers/hypixel/manifest.json#L9-L11

I think you are referring to the web_support line, not all of them.